### PR TITLE
Stl 726 doc fixes from stl 708

### PR DESCRIPTION
**Pull request by chenette**
Sawtooth doc changes for the 1.0 doc refresh, in separate commits (as much as possible).

### CODE REVIEW

File: docs/source/architecture/poet.rst [hunk 1]
```diff
@@ -21,6 +21,59 @@ use of a trusted execution environment, identity verification and blacklisting
 based on asymmetric key cryptography, and an additional set of election
 policies.
 
+.. START OF POET CONTENT MOVED FROM INTRODUCTION
```
Comment (jsmitchell) [inline docs/source/architecture/poet.rst]: Remove

File: docs/source/architecture/poet.rst [hunk 2]
```diff
@@ -21,6 +21,59 @@ use of a trusted execution environment, identity verification and blacklisting
...
+provided by other lottery algorithms. The probability of election
+is proportional to the resources contributed (in this case, resources
+are general purpose processors with a trusted execution environment).
+An attestation of execution provides information for verifying that the
+certificate was created within the enclave (and that the validator waited
+the allotted time). Further, the low cost of participation increases the
+likelihood that the population of validators will be large, increasing
+the robustness of the consensus algorithm.
+
+.. END OF POET CONTENT MOVED FROM INTRODUCTION
```
Comment (jsmitchell) [inline docs/source/architecture/poet.rst]: Remove

File: docs/source/architecture/journal.rst [hunk 3]
```diff
@@ -309,16 +314,14 @@ Block validation has the following steps that are always run in order. Failure
 of any validation step results in failure, processing is stopped, and the Block
 is marked as Invalid.
 
-1. ** Permission and Validation** - The following checks must succeed:
-
-   * On-chain transaction permissions are checked to see who is allowed to
-     submit transactions and batches.
+1. ** Transaction Permissioning ** - On-chain transaction permissions are
```
Comment (dcmiddle) [inline docs/source/architecture/journal.rst]: extra spaces between asterisks?

File: docs/source/architecture/journal.rst [hunk 4]
```diff
@@ -309,16 +314,14 @@ Block validation has the following steps that are always run in order. Failure
...
+   checked to see who is allowed to submit transactions and batches.
 
-   * On-chain block validation rules are checked to ensure that the Block
-     doesn't invalidate any of the on-chain validation rules stored at
-     ``sawtooth.validator.block_validation_rules``.
+#. ** On-chain Block Validation Rules ** - The on-chain block validation rules
```
Comment (dcmiddle) [inline docs/source/architecture/journal.rst]: extra spaces between asterisks?

File: docs/source/architecture/poet.rst (hunk 1 above)
Comment (chenette) [reply docs/source/architecture/poet.rst]: @jsmitchell - Double-checking: I should remove the two comments ("... START OF POET CONTENT" and "... END OF POET CONTENT") but not the actual PoET content, right? 

If so, will do.  And sorry - these were WIP comments to myself as I was moving text around, but I forgot to delete those comments.

File: docs/source/transaction_family_specifications/validator_registry_transaction_family.rst [hunk 5]
```diff
@@ -38,24 +38,22 @@ The following protocol buffers definition defines the validator info:
 .. code-block:: protobuf
 
   message ValidatorInfo {
```
[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/transaction_family_specifications/validator_registry_transaction_family.rst]: Sounds good to me, but I don't know how to do this.  Can anyone help?  I think an "include" directive is involved, but am not sure of the mechanics.

File: docs/source/architecture/poet.rst (hunk 1 above)
Comment (jsmitchell) [reply docs/source/architecture/poet.rst]: yep, the .. comments is what I was referring to

File: docs/source/transaction_family_specifications/validator_registry_transaction_family.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (dcmiddle) [reply docs/source/transaction_family_specifications/validator_registry_transaction_family.rst]: Take a look at the private UTXO doc.

[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/transaction_family_specifications/validator_registry_transaction_family.rst]: Aha - it's "literalinclude". Thanks!

[the remark this answers is not in the document]
Comment (chenette) [reply docs/source/transaction_family_specifications/validator_registry_transaction_family.rst]: @dan - I'd like to merge this PR's changes asap, so I've put this change on the to-do list for a later cleanup pass (non-technical edits and improvements). Thanks for this great suggestion!

**Pull request merged**